features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml

```diff
@@ -524,6 +524,25 @@
             </Read>
         </Scopes>
     </APIResourceCollection>
+    <APIResourceCollection name="tenants" displayName="Root Organizations" type="tenant">
+        <Scopes>
+            <Feature>
+                <Scope name="console:tenants"/>
+            </Feature>
+            <Update>
+                <Scope name="internal_modify_tenants"/>
+            </Update>
+            <Delete>
+                <Scope name="internal_modify_tenants"/>
+            </Delete>
+            <Create>
+                <Scope name="internal_list_tenants"/>
```

Comment:
`internal_list_tenants` is what we have in the identity.xml.
Hope this is intended @hwupathum 

https://github.com/wso2/carbon-identity-framework/blob/22b18dfcd8e7f94847bdf4bdc6ecef56bd7119ba/features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2#L3230-L3233

Reply:
Yes. This is the existing permission for all the GET methods in tenant management API

Reply:
There's `internal_create_tenants` scope but seems like its not being used in identity.xml.

https://github.com/wso2/carbon-identity-framework/blob/22b18dfcd8e7f94847bdf4bdc6ecef56bd7119ba/features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/system-api-resource.xml#L347C13-L348C67

Reply:
Yeah we should use `internal_create_tenants` scope

Reply:
Updated with https://github.com/wso2/carbon-identity-framework/pull/6047/commits/8a50a36f535996eb91db9c18642ae1231f07a080

We should update the `identity.xml` as well right @hwupathum ?

Reply:
Yes, to be used with legacy runtime.